release-tools/stochss_ami_manager.py

```diff
@@ -70,6 +70,7 @@ class AmiManager:
     NUM_TRIALS = 5
     NUM_SSH_TRIALS = 10
     DEFAULT_AMI_FILENAME = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'ami.json')
+    DEFAULT_STOCHSS_AMI_FILENAME = os.path.join(os.path.abspath(os.path.dirname(__file__)), '../conf/ec2_config.json')
```

Comment:
Better to do os.path.abspath(os.path.dirname(__file__)), '..', 'conf', 'ec2_config.json')